.github/workflows/build.yml

```diff
@@ -98,7 +108,6 @@ jobs:
 
   integration-tests:
     name: Integration tests
```

Comment:
There's no need to have the dependency on the build here since it's installing everything it needs